### ISSUE TITLE
Some comments

### ISSUE THREAD

**Issue by quanpn90** [FlorianPfisterer/2d-seq2seq#1]
Hi, 

Can I discuss something regarding the implementation by creating issues here ? (OFc we can use another communication channel if you prefer). 

- Why do you permute the output size with permute(0, 2, 1). Typically the label should have size (batch_size, sequence_length) or (sequence_length, batch_size). So afai understand your output (y_pred) should have already had the size (sequence_len x batch x output_vocab_size) which should be correct already. 
- The CrossEntropyLoss module includes LogSoftMax and NLL (Negative Likelihood loss), so the softmax layer in the LSTM2D should be removed.
- What worries me the most is to handle padding in this network (sentences grouped in the same mini-batch normally have different lengths, so they are padded). The input to the cell has two directions so it could be more annoying than normal LSTM.

Thanks for your work! 
Quan
